app/views/admin/display_ads/_form.html.erb

```diff
@@ -17,18 +17,19 @@
 
     <div class="crayons-field">
       <%= label_tag :placement_area, "Placement Area:", class: "crayons-field__label" %>
-      <%= select_tag :placement_area, options_for_select(display_ads_placement_area_options_array, selected: @display_ad.placement_area), include_blank: "Select...", class: "crayons-select" %>
+      <%= select_tag :placement_area, options_for_select(display_ads_placement_area_options_array, selected: @display_ad.placement_area), include_blank: "Select...", class: "crayons-select js-placement-area" %>
     </div>
 
     <% if FeatureFlag.enabled?(:display_ad_tags) %>
-      <div class="crayons-field">
-        <div id="display-ad-targeted-tags"></div>
-      </div>
 
-      <div class="crayons-field hidden">
-        <%= label_tag :tag_list, "tag_list:", class: "crayons-field__label" %>
-        <%= text_field_tag :tag_list, @display_ad.tag_list, class: "crayons-textfield js-tags-textfield" %>
-      </div>
+        <div class="crayons-field">
+          <div id="display-ad-targeted-tags"></div>
+        </div>
+
+        <div class="crayons-field hidden">
+          <%= label_tag :tag_list, "tag_list:", class: "crayons-field__label" %>
```

Comment:
Non-blocking question: Is the label `tag_list:` here a placeholder?

Reply:
Nope, sorry I made a typo on the name of the field and I didn't notice the typo coz its a hidden field. 

This field is basically responsible for showing the selections from the MultiSelectAutocomplete component so that it can be saved on the Display Ad model as tag_list.

Reply:
Should have been <%= label_tag :tag_list, "Tag List:", class: "crayons-field__label" %>